neurips2022/track2/train/utility_IL.py

```diff
@@ -63,16 +63,21 @@ def load_data_scratch(dataset_path, save_path):
                     vehicle_ids.append(vehicle_id)
 
         for id in vehicle_ids:
+            for filename in os.listdir(os.path.join(path, scenario)):
+                match = re.search(r"{}\.pkl".format(id), filename)
+                if match is not None:
+                    pkl_file = match
```

Comment:
What is the function of this code?

Reply:
To find a specific vehicle_id's pkl file.